### PR TITLE
TASK: Update constraints

### PR DESCRIPTION
**Pull request by dlubitz**
Add fusion as contstraint to this package.

I set the constraint more tight, than needed. But I didn't want to cover outdated/unsupported versions.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -4,7 +4,8 @@
     "description": "Flow Form Framework preset for Fusion based Form rendering",
     "license": "MIT",
     "require": {
-        "neos/form": "^5.0"
+        "neos/form": "^5.0",
+        "neos/fusion": ">=7.3"
```
Comment (bwaidelich) [inline composer.json]: why didn't you use semantic version constraints, e.g. (`^7.3`)?

Comment (dlubitz) [reply composer.json]: Yeah, than we need to change it with each Neos version.

Comment (kdambekalns) [reply composer.json]: And that makes sense! Never use unlimited constraints!
```suggestion
        "neos/fusion": "^7.3 || ^8.3 || ^9.0"
```

Comment (dlubitz) [inline composer.json:8]:
```suggestion
        "neos/fusion": "^7.3 || ^8.0 || ^9.0"
```

"^8.0" right?

Comment (dlubitz) [reply composer.json]:
```suggestion
        "neos/fusion": "^7.3 || ^8.0 || ^9.0"
```

"^8.0" right?

Comment (kdambekalns) [reply composer.json]: 🙈 Yes, of course!

**Review by kdambekalns — APPROVED** [review]

**Pull request merged**